lib/resources/datav2/entityv2.ts

```diff
@@ -384,3 +409,18 @@ function convertSnapshotData(key: string, data: any) {
       return data;
   }
 }
+
+function convertCryptoSnapshotData(key: string, data: any) {
+  switch (key) {
+    case "LatestTrade":
+      return AlpacaCryptoTrade(data);
+    case "LatestQuote":
+      return AlpacaCryptoQuote(data);
+    case "MinuteBar":
+    case "DailyBar":
+    case "PrevDailyBar":
+      return AlpacaCryptoBar(data);
+    default:
+      return data;
```

Comment:
Write a new function for this is not necessary, you can easily extend the existing one like this: 
```
function convertSnapshotData(key: string, data: any, isCrypto: boolean) {
  switch (key) {
    case "LatestTrade":
      return isCrypto ? AlpacaCryptoTrade(data) : AlpacaTradeV2(data);
    case "LatestQuote":
      return isCrypto ? AlpacaCryptoQuote(data) : AlpacaQuoteV2(data);
    case "MinuteBar":
    case "DailyBar":
    case "PrevDailyBar":
      return isCrypto ? AlpacaCryptoBar(data) : AlpacaBarV2(data);
    default:
      return data;
  }
}
```